process-exporter/daemonset.yaml

```diff
@@ -14,6 +14,9 @@ spec:
       name: process-exporter
       namespace: openshift-monitoring
     spec:
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          effect: NoSchedule
```

Comment:
This won't work well for SNO.

Can we switch this around so it's opt-in (schedule on everything tagged with `node-role.kubernetes.io/worker`)?

Reply:
Yes, but doesn't SNO roles are master&worker?

Reply:
Yes, SNO is tagged with both.  So won't this, as written, prevent scheduling on SNO because it is a `master`?

Maybe for now just run it everywhere?

Reply:
As you did in here?
```yaml
tolerations:
        - operator: Exists
```